### ISSUE TITLE
[torchlib] `_attention_scale` generates function_proto that breaks SSA

### ISSUE THREAD

**Issue by BowenBao** [microsoft/onnxscript#848]
```python
from onnxscript.function_libs.torch_lib.ops.nn import _attention_scale
# Look for node `Gather`, input and output are both 'tmp'.
"""
node {
  input: "tmp"
  input: "int64_m1"
  output: "tmp"
  name: "n2"
  op_type: "Gather"
  attribute {
    name: "axis"
    i: 0
    type: INT
  }
  domain: ""
}
"""
print(_attention_scale.to_function_proto())
```

**Comment by gramalingam** [comment]
I suspect it must be from the extension I merged in for translation of `X[Y]`. I will take a look.

I wonder if we should have test-cases that check the validity of function-translation in the CI.

**Comment by BowenBao** [comment]
Thanks @gramalingam. In fact I'm surprised it is not caught. I think the ort fullgraph test under ops_test.py should run with model_proto generated from it. Maybe the branch uses `_attention_scale` is not covered? @justinchuby

**Comment by justinchuby** [comment]
Not seeing crashes in my latest tests. Need to verify.

**Comment by BowenBao** [comment]
I believe OP is resolved by #851. We can close this and track enabling `nn.functional.scaled_dot_product_attention` in CI separately.
